release-info/introduction.md

```diff
@@ -39,8 +39,8 @@ a new minor version of OCaml is released every six months.
 
 For instance, at the date of writing, the next planned releases of OCaml are:
 
-- OCaml 5.2: around April 2024
 - OCaml 5.3: around October 2024
+- OCaml 5.4: around April 2025
```

Comment:
Could you make this document time-insensitive by pointing to your new file? Is this a good idea?

Reply:
I think that you should either accept that this is an example that can be out of date, and stop updating it, or accept that you want it to be up to date, and refer to the other file.

Reply:
Sure, I shall try to accept that the example can be out-of-date next time.